modules/travel_pay/app/controllers/travel_pay/v0/documents_controller.rb

```diff
@@ -42,6 +79,28 @@ def handle_resource_not_found_error(e)
           status: :not_found
         )
       end
+
+      def verify_feature_flag_enabled!
+        return if Flipper.enabled?(:travel_pay_enable_complex_claims, @current_user)
+
+        message = 'Travel Pay expense submission unavailable per feature toggle'
+        Rails.logger.error(message:)
+        raise Common::Exceptions::ServiceUnavailable, message:
+      end
+
+      def validate_claim_id_exists!(claim_id)
+        # NOTE: In request specs, you can’t make params[:claim_id] truly missing because
+        # it’s part of the URL path and Rails routing prevents that.
+        raise Common::Exceptions::BadRequest.new(detail: 'Claim ID is required') if claim_id.blank?
+
+        raise Common::Exceptions::BadRequest.new(detail: 'Claim ID is invalid') unless claim_id =~ /\A[\w-]+\z/
```

Comment:
This may be a little too lenient compared to what we have in other places (see https://github.com/department-of-veterans-affairs/vets-api/blob/6688427c16acef93463b7ab6509ec41e730a14cc/modules/travel_pay/app/services/travel_pay/claims_service.rb#L54)

Can we keep the claim ID regex consistent?